master/proto/api/v1/api.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package api.v1;
+option go_package = "github.com/determined-ai/determined/master/pkg/proto/apiv1";
```

Comment:
question: is `apiv1` rather than `api/v1` intentional?

doesn't seem clear to me why we are doing this

Reply:
It is, this is standard for versioned go_packages. Unfortunately, it will produce illegal go code as a result because it generates conflicting package names.

Reply:
ahhh I see now